Accepted-RFCs/RFC-013-new-initiative-process.md

```diff
@@ -0,0 +1,124 @@
+# RFC-013: New initiative process
+
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+
+Initially submitted on 24 Aug 2022
+
+## Current status
+
+- [x] Draft
+- [ ] Under discussion
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview
+
+The Good Docs Project has an issue of delivery: we haven't delivered a 1.0 of our templates since the inception. And as a volunteer org, we need to keep focus on our mission and goal. It's easy to dilute ourselves if we don't keep ourselves accountable to focus and intentional communal growth.
+
+In other words, we as a project need to make sure we can at least do our core mission well. All new initiatives should 
+
+(As a personal note, the drafter of this RFC has ADHD, so "new project energy" is a real thing, a true risk of distraction and effort dilution.)
+
+
+## Motivation
+
+As we grow, our risk of project scope creep and sprawl grows. As a group, we should grow *with intentionality*. 
+
+
+## Proposal
+
+When someone wants to start a new working group or sub-project in the Good Docs Project, it should coherently address at least one of:
```

Comment:
After reading the initial set of criteria I had this thought:

It would be nice to have the criteria in check-list form. That could make a  nice "cover sheet" for new repos until they write a proper README.

Reply:
Noted in proposal